roofit/roofitcore/test/testRooFuncWrapper.cxx

```diff
@@ -21,11 +21,16 @@
 #include <RooFitResult.h>
 #include <RooFuncWrapper.h>
 #include <RooGaussian.h>
+#include <RooHistPdf.h>
 #include <RooHelpers.h>
 #include <RooMinimizer.h>
+#include <RooPoisson.h>
+#include <RooPolynomial.h>
 #include <RooRealVar.h>
+#include <RooRealSumPdf.h>
 #include <RooSimultaneous.h>
 #include <RooWorkspace.h>
+#include "RooStats/HistFactory/ParamHistFunc.h"
```

Comment:
```suggestion
#include <RooStats/HistFactory/ParamHistFunc.h>
```
To be consistent with the rest of the file.

I tend to use `<>` for headers that will get installed, and `""` for the private headers that don't get installed.